client/src/document/index.scss

```diff
@@ -46,11 +58,6 @@
     color: var(--text-link);
     width: fit-content;
 
```

Comment:
```suggestion

    text-decoration: underline;

    &:hover,
    &:focus {
      text-decoration: none;
    }
```

Reply:
@schalkneethling can do the change here. I'm not sure where we use `.button` but changing the text-decoration here seems more defensive with the same result.

Reply:
Yeah, I was going to do it there, but then I was unsure of the down stream impact as I did not add that piece of CSS. Happy to make the change there instead though. Will update the PR.

Reply:
Now I remember the problem I ran into. If I added it to the `a:not(.button) {` block then it also underlines all the headings which I believe is not what we want. Because of the specificity of `a:not(.button)` it overrides what is set on line `11+` even if I move it to be before these in the CSS.

Am I missing something?